modules/apps/bulk/bulk-selection-test/src/testIntegration/java/com/liferay/bulk/selection/test/util/TestBulkSelectionFactory.java

```diff
@@ -31,9 +31,7 @@
 /**
  * @author Alejandro Tardín
  */
-@Component(
-	service = BulkSelectionFactory.class
-)
+@Component(service = BulkSelectionFactory.class)
```

Comment:
Hey @RenanUmV, this change must be into this commit https://github.com/liferay-core-infra/liferay-portal/pull/4522/commits/b7cd433b95ea4b949161a4cf94f91c8a8792493a